.gitmodules

```diff
@@ -4,3 +4,4 @@
 [submodule "crates/contracts/lib/eigenlayer-middleware"]
 	path = crates/contracts/lib/eigenlayer-middleware
 	url = https://github.com/Layr-labs/eigenlayer-middleware
+    commit = 512ce7326f35e8060b9d46e23f9c159c0000b546
```

Comment:
this commit is for rewards i think

Reply:
Good catch! I discarded all changes on `crates/contracts/lib` but forgot this file was here 😅